eval/ph/data_collection.py

```
@@ -0,0 +1,289 @@
+    arr = np.asarray(values, dtype=np.float64)
+    return float(arr.mean()), float(arr.std(ddof=0))
+
+
+def collect_pipeline_results(
+    setups: list[Setup],
+    config: OptimizationConfig | None = None,
+    num_unitaries_per_setup: int = 10,
+    repeats_per_unitary: int = 3,
+    phase_errors: Iterable[float] = (0.01,),
+    base_seed: int = 0,
+    input_losses: bool = False,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)